app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -195,9 +244,31 @@ You can contact your teacher training provider to check:
 
 * the content of your teacher training course meets the 50% requirement
 
-### 2. Type of visa for trainees
+### 2. Degree requirements for trainees
+
+If you are starting your teacher training course in or after September 2024, you must have studied for your undergraduate (bachelor’s) degree at a [university recognised](https://www.enic-naric.net/page-recognised-heis) by the European Network of Information Centres (ENIC).
```

Comment:
Need to tweak this link text to make it more accessible. 
```suggestion
If you are starting your teacher training course from September 2024, you must have studied for your undergraduate (bachelor’s) degree at a university recognised by the European Network of Information Centres (ENIC). You can check if your university is recognised on the [ENIC website](https://www.enic-naric.net/page-recognised-heis).
```

Reply:
We decided to remove this as it's not quite accurate.